src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -139,7 +140,16 @@ export class TerminalChatWidget extends Disposable {
 				}
 			},
 		);
-		this._register(this._inlineChatWidget.chatWidget.onDidChangeViewModel(() => this._saveViewState()));
+		this._register(autorun(r => {
```

Comment:
I think we can delete `acceptInput` now?

Reply:
probably need the history stuff from that and maybe other things too?

Reply:
![Screenshot 2024-11-06 at 4 36 58 PM](https://github.com/user-attachments/assets/95e05d90-8953-49de-9882-0d0ff0aa0456)
that's still used

Reply:
https://github.com/microsoft/vscode/blob/2cac90058f0df43c91dde33bb2b8910720529368/src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts#L47